### PR TITLE
add travis-ci tool

### PR DESCRIPTION
**Pull request by andreaturli**

### CODE REVIEW

**Comment by grkvlt** [comment]
*JDK

File: .buildscript/deploy_snapshot.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  echo "Skipping snapshot deployment: wrong repository. Expected '$SLUG' but was '$TRAVIS_REPO_SLUG'."
+elif [ "$TRAVIS_JDK_VERSION" != "$JDK" ]; then
+  echo "Skipping snapshot deployment: wrong JDK. Expected '$JDK' but was '$TRAVIS_JDK_VERSION'."
+elif [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
+  echo "Skipping snapshot deployment: was pull request."
+elif [ "$TRAVIS_BRANCH" != "$BRANCH" ]; then
+  echo "Skipping snapshot deployment: wrong branch. Expected '$BRANCH' but was '$TRAVIS_BRANCH'."
+else
+  echo "Deploying snapshot..."
+  mvn clean source:jar javadoc:jar deploy --settings=".buildscript/settings.xml" -Dmaven.test.skip=true
```
Comment (andreaturli) [inline .buildscript/deploy_snapshot.sh]: @grkvlt you mean I can remove source:jar and javadoc:jar here?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+sudo: false
+language: java
+jdk:
+  - oraclejdk8
```
Comment (andreaturli) [inline .travis.yml]: I'll try by removing the jdk from here but not sure it mvn needs it anyway

Comment (andreaturli) [inline .travis.yml]: @grkvlt I've removed java requirements and it looks fine

File: .buildscript/deploy_snapshot.sh (hunk 1 above)
Comment (grkvlt) [inline .buildscript/deploy_snapshot.sh]: Yes

File: .travis.yml (hunk 2 above)
Comment (grkvlt) [inline .travis.yml]: Great

File: .buildscript/deploy_snapshot.sh [hunk 3]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#
+# Deploy a jar, source jar, and javadoc jar to Sonatype's snapshot repo.
```
Comment (grkvlt) [inline .buildscript/deploy_snapshot.sh]: Change comment as well, to remove Java references

**Comment by andreaturli** [comment]
@grkvlt I think I've addressed your comments. Good to merge?

**Pull request merged**